src/components/pages/StatisticsPage/StatisticsContainer/StatisticsContainer.tsx

```diff
@@ -0,0 +1,20 @@
+import { ResultNumberField } from "../../ResultQuizPage/ResultNumberField/ResultNumberField";
+import { StatisticsContainerProps } from "./interface/StatisticsContainer.interface";
+
+const StatisticsContainer = ({ data }: StatisticsContainerProps) => {
+    return (
+        <>
+            {
+                Object.entries(data).map(item => (
+                    <ResultNumberField
+                        key={item[0]}
+                        text={item[0]}
+                        value={item[1]}
+                    />
+                ))
+            }
+        </>
+    );
+}
```

Comment:
```suggestion
    return Object.entries(data).map(item => (
                    <ResultNumberField
                        key={item[0]}
                        text={item[0]}
                        value={item[1]}
                    />
                ));
}
```
there is no need to wrap it with fragment :)